python/dllib/src/bigdl/dllib/keras/Sequential.py

```diff
@@ -20,12 +20,13 @@
 from .engine.topology import KerasNet
```

Comment:
move this to `models.py`, so that we can have both `dllib.keras.Sequential` and `dllib.keras.models.Sequential`